includes/class-wc-stripe-blocks-support.php

```diff
@@ -515,14 +515,22 @@ public function add_stripe_intents( PaymentContext $context, PaymentResult &$res
 	 * @param string    $payment_request_type The payment request type used for payment.
 	 */
 	private function add_order_meta( \WC_Order $order, $payment_request_type ) {
-		if ( 'apple_pay' === $payment_request_type ) {
-			$order->set_payment_method_title( 'Apple Pay (Stripe)' );
-			$order->save();
-		} elseif ( 'google_pay' === $payment_request_type ) {
-			$order->set_payment_method_title( 'Google Pay (Stripe)' );
-			$order->save();
-		} elseif ( 'payment_request_api' === $payment_request_type ) {
-			$order->set_payment_method_title( 'Payment Request (Stripe)' );
+		$payment_method_title = '';
+		switch ( $payment_request_type ) {
+			case 'apple_pay':
+				$payment_method_title = WC_Stripe_Payment_Methods::APPLE_PAY_LABEL;
+				break;
+			case 'google_pay':
+				$payment_method_title = WC_Stripe_Payment_Methods::GOOGLE_PAY_LABEL;
+				break;
+			case 'payment_request_api':
+				$payment_method_title = 'Payment Request';
```

Comment:
Should we move "Payment Request" to a constant as well?

Reply:
Sure! Did it in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3844/commits/2454e16de799a32e341aefc00d1568200c97df4e